app/assets/javascripts/discourse/app/widgets/topic-map.js

```diff
@@ -87,12 +87,15 @@ createWidget("topic-participant", {
         linkContents.push(this.attach("avatar-flair", autoFlairAttrs));
       }
     }
-
     return h(
       "a.poster.trigger-user-card",
       {
         className: state.toggled ? "toggled" : null,
-        attributes: { title: attrs.username, "data-user-card": attrs.username },
+        attributes: {
+          title: attrs.username,
+          "data-user-card": attrs.username,
+          href: `/u/${attrs.username}`,
```

Comment:
I _think_ it's safer to use the `userPath` helper here from `discourse/lib/url`.

Reply:
Safer for edge cases like subfolder sites, I mean.

Reply:
Cool! Wasn't aware this existed, thanks :) Pushed an update 🙌🏽